src/java/org/apache/cassandra/index/sai/disk/format/IndexDescriptor.java

```diff
@@ -439,7 +449,8 @@ public ChecksumIndexInput openCheckSummedPerIndexInput(IndexComponent component,
      */
     private ChecksumIndexInput checksumIndexInput(IndexContext context, IndexInput indexInput)
     {
-        return getVersion(context) == Version.AA
+        Version version = getVersion(context);
+        return version == Version.AA || version == Version.AB
```

Comment:
should we fail in case of other index version start from A or accept all index versions starting from A for the first case?